common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +54,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    traceMetadata: Option[SpanMetadata] = None)
```

Comment:
@ddragosd in that case there are no fields sent. Here is a sample JSON in case Tracing is not enabled or its enabled but trace has not been collected based on sampling rate. As you can notice in this case there will be no Tracemetadata key present in message.
`{"activationId":"4b50591d891f4769b1f40917fc61af9f","revision":"1-61cb0057d6b1cddd315f1998da518ca3","activationNamespace":"guest","transid":[28,1496053775867],"content":{},"action":{"path":"guest","name":"hello","version":"0.0.1"},"user":{"subject":"guest","namespace":"guest","authkey":"23bc46b1-71f6-4ed5-8c54-816aa4f8c502:123zO3xZCLrMN6v2BKK1dXYFpXlPkccOFqm12CdAsMgRU4VrNZ9lyGVCGuMDGIwP","rights":["READ","PUT","DELETE","ACTIVATE"]}}`